app/src/main/java/gov/hhs/cdc/trustedintermediary/external/localfile/LocalSecrets.java

```diff
@@ -27,17 +29,37 @@ public static LocalSecrets getInstance() {
     @Override
     public String getKey(String secretName) throws SecretRetrievalException {
 
-        logger.logInfo("Acquiring local key...");
+        logger.logInfo("Acquiring local key " + secretName);
         String key = "";
 
         try {
-            key = Files.readString(Path.of("..", "mock_credentials", secretName + ".pem"));
-        } catch (IOException e) {
-            var message = "Error getting local key";
-            logger.logError(message, e);
-            throw new SecretRetrievalException(message, e);
+            key = readSecretFromFileSystem(secretName);
+        } catch (SecretRetrievalException exception) {
+            logger.logInfo(
```

Comment:
I think logError would be more appropiate here

Reply:
I decided to do the info level because this may not be an error and completely OK because the secret may be in the resources instead on the filesystem.  Maybe this should be at the warn level?

Reply:
Ok, that's fair. I'd vote to log it as warning or debug as I see INFO more as log for normal/expected flow, but I don't mind if left as is

Reply:
I changed it to a warning.